packages/react-components/react-toast/stories/Toast/CustomTimeout.stories.tsx

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
 import { Toaster, useToastController } from '@fluentui/react-toast';
+import { useId } from '@fluentui/react-components';
 
 export const CustomTimeout = () => {
+  const toasterId = useId('toaster');
   const { dispatchToast } = useToastController();
-  const notify = () => dispatchToast('This is a toast', { timeout: 1000 });
+  const notify = () => dispatchToast('This is a toast', { timeout: 1000, toasterId });
 
   return (
     <>
-      <Toaster />
+      <Toaster toasterId={toasterId} />
```

Comment:
All examples need to use `toasterId` because all examples are rendered to one docpage - multi-toaster scenario by default